spec/amp-html-layout.md

```diff
@@ -68,6 +68,8 @@ the layout of a `fill` element matches its parent.
 
 The element lets its children to define its size, much like a normal HTML `div`.
 
+### `flex-item`
+The element and other elements with layout type `flex-item` take their parent container's remaining space when parent has display : flex. 
```

Comment:
1. What are "and other elements"?
2. "display : flex" -> `display: flex` with apostrophe quotes.